server/server.ts

```diff
@@ -3,19 +3,19 @@ import * as path from "https://deno.land/std/path/mod.ts";
 import router from "./routes.ts";
 
 const PORT = 8000;
-const HOST = "localhost";
+//const HOST = "localhost";
```

Comment:
This comment is necessary?

Reply:
no, but before my change it was used to start server, but it only started server on 127.0.0.1 and not on local network. After change it starts on both. So I can delete this comment.